src/zenml/zen_stores/migrations/alembic.py

```diff
@@ -34,6 +34,7 @@
 from zenml.zen_stores import schemas
 
 ZENML_ALEMBIC_START_REVISION = "alembic_start"
+DATABASE_LOCK_TIMEOUT = 15
```

Comment:
Is this long enough? Wonder whether it should be increased to 30 just in case?

Reply:
I think it is, it was even working with 5s